log/gps/gps_destinator_trk.ksy

```diff
@@ -0,0 +1,82 @@
+meta:
+  id: gps_destinator_trk
+  title: Destinator track
+  application: Destinator
+  file-extension:
+    - dat
+  license: GPL-2.0-or-later
+  encoding: utf-16le
+  endian: le
+
+doc: |
+  Destinator formats
+
+doc-ref:
+  - https://web.archive.org/web/20160310115940/http://mozoft.com/d3log.html
+  - https://web.archive.org/web/20071011022251/http://www.destinatortechnologies.net/uk/products/products/bundles/
+  - https://www.gpsbabel.org/htmldoc-development/fmt_destinator_trl.html
+  - https://github.com/gpsbabel/gpsbabel/blob/e1e195beca8e0f24ca0e41c02d1986af13865cd0/destinator.cc#L264L326
+
+
+seq:
+  - id: points
+    type: point
+    repeat: eos
+
+types:
+  point:
+    doc: |
+      152 bytes
+      For 3.0.75_103, PN 5.1.80 and 6.0.0.24556
```

Comment:
I assume these are the versions of the Destinator software that were used to test this spec? For clarity it would be helpful to say that in the `doc`.